src/app/infrastructure/colorUtils.ts

```diff
@@ -0,0 +1,14 @@
+import * as Color from 'color';
```

Comment:
Cool idea 👍

Reply:
lol, not my idea, I've just used it in the past. 
I mean't to put a link to where I got it from.  Added it.